sp1up/sp1up

```diff
@@ -80,12 +80,12 @@ main() {
 
   # Install by downloading binaries
   if [[ "$SP1UP_REPO" == "succinctlabs/sp1" && -z "$SP1UP_BRANCH" && -z "$SP1UP_COMMIT" ]]; then
-    SP1UP_VERSION=${SP1UP_VERSION:-nightly}
+    SP1UP_VERSION=${SP1UP_VERSION:-main}
     SP1UP_TAG=$SP1UP_VERSION
 
-    # Normalize versions (handle channels, versions without v prefix
-    if [[ "$SP1UP_VERSION" =~ ^nightly ]]; then
-      SP1UP_VERSION="nightly"
+    # Normalize versions (handle channels, versions without v prefix)
+    if [[ "$SP1UP_VERSION" == "main" ]]; then
+      SP1UP_VERSION="main"
```

Comment:
not sure if this fully makes sense?

Reply:
Yeah agreed, will clean this up. Before it just used the latest tag that started with `nightly`.

Reply:
Note: If we change back to nightly releases @jtguibas we'll need to re-add this.